documentation/graphing-overview/index.html.md

```diff
@@ -0,0 +1,279 @@
+---
+title: "Flow.js and Data Visualization: Graphing with Contour"
+layout: "flow"
+isPage: true
+---
+
+##Flow.js and Data Visualization: Graphing with Contour
+
+
+Flow.js makes it easy to show data from your model variables in [text](../generated/dom/attributes/binds/default-bind-attr/) or [lists](../generated/dom/attributes/foreach/default-foreach-attr/). However, often you want more than just textual output &mdash; you want to visualize the results of your simulation!
+
+One way to do this is to use <a href="http://forio.com/contour/" target="_blank">Contour</a>, an open source JavaScript library developed by Forio for interactive data visualization, based on <a href="http://d3js.org" target="_blank">d3.js</a>. Most of the [example projects](../../project_admin/#personal) in Epicenter use Contour for their graphs.
```

Comment:
sadly no, at least not through docpad (which requires html) or jekyll (used for the forio.com marketing site. although with jekyll you can add a "{:target="_blank"}" at the end ... but that doesn't work for docpad.)